Too many open files (os error 24) on macos

**Summary**
It seems that this can be solved by changing the maxfile, but maybe we should reduce test threads number (all tests run in the same tokio runtime?)



```
test pipelines::transforms::transform_limit_by_test::test_transform_limit_by ... ok
test pipelines::transforms::transform_filter_test::test_transform_filter_error ... ok
test pipelines::processors::pipeline_display_test::test_pipeline_display ... ok
test pipelines::processors::pipeline_walker_test::test_pipeline_walker ... ok
test pipelines::processors::pipeline_builder_test::test_local_pipeline_builds ... ok
test pipelines::transforms::transform_source_test::transform_source_test ... ok
test pipelines::transforms::transform_projection_test::test_transform_projection ... ok
test servers::http::v1::block_to_json_test::test_data_block_not_nullable ... ok
test servers::http::v1::block_to_json_test::test_data_block_nullable ... ok
test pipelines::transforms::transform_sort_test::test_transform_sort ... ok
test servers::clickhouse::clickhouse_handler_test::test_abort_clickhouse_server ... ok
test servers::mysql::mysql_handler_test::test_rejected_session_with_sequence ... FAILED
test sessions::session_test::test_session_mem_usage ... ok
test sql::parser::ast::ast_display_test::test::test_display_create_database ... ok
test sql::parser::ast::ast_display_test::test::test_display_create_table ... ok
test sql::parser::ast::ast_display_test::test::test_display_expr ... ok
test sql::parser::ast::ast_display_test::test::test_display_query ... ok
test sql::parser::ast::ast_display_test::test::test_display_table_reference ... ok
test sql::parser::transformer::transform_sqlparser_test::test::test_parse_with_ddl ... ok
test sql::parser::transformer::transform_sqlparser_test::test::test_parse_with_sqlparser ... ok
test optimizers::optimizer_constant_folding_test::tests::test_constant_folding_optimizer ... FAILED
test pipelines::transforms::transform_limit_test::test_transform_limit ... FAILED
test sql::sql_parser_test::create_database ... ok
test sql::sql_parser_test::create_table ... ok
test servers::mysql::mysql_handler_test::test_use_database_with_on_query ... FAILED
test sql::sql_parser_test::describe_table ... ok
test optimizers::optimizer_expression_transform_test::tests::test_expression_transform_optimizer ... FAILED
test sql::sql_parser_test::hint_test ... ok
test sql::sql_parser_test::drop_database ... ok
test sql::sql_parser_test::drop_table ... ok
test sql::sql_parser_test::truncate_table ... ok
test sql::sql_parser_test::show_tables_test ... ok
test sql::sql_parser_test::show_databases_test ... ok
test sql::sql_parser_test::use_database_test ... ok
test sql::sql_parser_test::show_queries ... ok
test servers::mysql::mysql_handler_test::test_rejected_session_with_parallel ... ok
test users::user_mgr_test::test_user_manager ... ok
test servers::http::v1::statement_test::test_statement ... ok
test sql::plan_parser_test::test_plan_parser ... ok
test servers::clickhouse::clickhouse_handler_test::test_clickhouse_handler_query ... ok
test servers::clickhouse::clickhouse_handler_test::test_clickhouse_insert_data ... ok
test optimizers::optimizer_scatters_test::test_scatter_optimizer ... ok
test datasources::database::system::credits_table_test::test_credits_table ... ok
test servers::clickhouse::clickhouse_handler_test::test_reject_clickhouse_connection ... ok

failures:

---- servers::mysql::mysql_handler_test::test_rejected_session_with_sequence stdout ----
Error: Code: 1001, displayText = Too many open files (os error 24).

   0: backtrace::backtrace::libunwind::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/libunwind.rs:93:5
      backtrace::backtrace::trace_unsynchronized
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:66:5
      backtrace::backtrace::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:53:14
   1: backtrace::capture::Backtrace::create
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:176:9
      backtrace::capture::Backtrace::new
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:140:22
   2: common_exception::exception::ErrorCode::TokioError
             at /Users/hxl/Desktop/code/myself/databend/common/exception/src/exception.rs:110:77
   3: common_base::runtime::Runtime::create::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:158:36
   4: core::result::Result<T,E>::map_err
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:853:27
   5: common_base::runtime::Runtime::create
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:156:23
   6: common_base::runtime::Runtime::with_worker_threads
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:202:9
   7: <databend_query::servers::mysql::mysql_handler::MySQLHandler as databend_query::servers::server::Server>::start::{{closure}}
             at src/servers/mysql/mysql_handler.rs:137:44
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
   9: <core::pin::Pin<P> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/future.rs:119:9
  10: databend_query::servers::mysql::mysql_handler_test::test_rejected_session_with_sequence::{{closure}}
             at src/servers/mysql/mysql_handler_test.rs:57:21
  11: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  12: tokio::park::thread::CachedParkThread::block_on::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:54
  13: tokio::coop::with_budget::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:106:9
  14: std::thread::local::LocalKey<T>::try_with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:399:16
  15: std::thread::local::LocalKey<T>::with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:375:9
  16: tokio::coop::with_budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:99:5
      tokio::coop::budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:76:5
      tokio::park::thread::CachedParkThread::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:31
  17: tokio::runtime::enter::Enter::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/enter.rs:151:13
  18: tokio::runtime::thread_pool::ThreadPool::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/thread_pool/mod.rs:77:9
  19: tokio::runtime::Runtime::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/mod.rs:463:43
  20: databend_query::servers::mysql::mysql_handler_test::test_rejected_session_with_sequence
             at src/servers/mysql/mysql_handler_test.rs:80:5
  21: databend_query::servers::mysql::mysql_handler_test::test_rejected_session_with_sequence::{{closure}}
             at src/servers/mysql/mysql_handler_test.rs:52:7
  22: core::ops::function::FnOnce::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  23: core::ops::function::FnOnce::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
      test::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:578:5
  24: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
      std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
      test::run_test_in_process
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:601:18
      test::run_test::run_test_inner::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:493:39
  25: test::run_test::run_test_inner::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:520:37
      std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys_common/backtrace.rs:125:18
  26: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:481:17
      <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
      std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
      std::thread::Builder::spawn_unchecked::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:480:30
      core::ops::function::FnOnce::call_once{{vtable.shim}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  27: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys/unix/thread.rs:106:17
  28: __pthread_start

thread 'servers::mysql::mysql_handler_test::test_rejected_session_with_sequence' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:194:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- optimizers::optimizer_constant_folding_test::tests::test_constant_folding_optimizer stdout ----
Error: Code: 1001, displayText = Too many open files (os error 24).

   0: backtrace::backtrace::libunwind::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/libunwind.rs:93:5
      backtrace::backtrace::trace_unsynchronized
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:66:5
      backtrace::backtrace::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:53:14
   1: backtrace::capture::Backtrace::create
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:176:9
      backtrace::capture::Backtrace::new
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:140:22
   2: common_exception::exception::ErrorCode::TokioError
             at /Users/hxl/Desktop/code/myself/databend/common/exception/src/exception.rs:110:77
   3: common_base::runtime::Runtime::create::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:158:36
   4: core::result::Result<T,E>::map_err
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:853:27
   5: common_base::runtime::Runtime::create
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:156:23
   6: common_base::runtime::Runtime::with_worker_threads
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:202:9
   7: databend_query::sessions::context_shared::DatabendQueryContextShared::try_get_runtime
             at src/sessions/context_shared.rs:143:40
   8: databend_query::sessions::context::DatabendQueryContext::get_shared_runtime
             at src/sessions/context.rs:225:9
   9: databend_query::sessions::context::DatabendQueryContext::get_single_node_table_io_context
             at src/sessions/context.rs:239:13
  10: databend_query::sql::plan_parser::PlanParser::plan_with_dummy_source::{{closure}}::{{closure}}
             at src/sql/plan_parser.rs:723:38
  11: core::result::Result<T,E>::and_then
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:966:22
  12: databend_query::sql::plan_parser::PlanParser::plan_with_dummy_source::{{closure}}
             at src/sql/plan_parser.rs:718:13
  13: core::result::Result<T,E>::and_then
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:966:22
  14: databend_query::sql::plan_parser::PlanParser::plan_with_dummy_source
             at src/sql/plan_parser.rs:706:9
  15: databend_query::sql::plan_parser::PlanParser::plan_tables_with_joins
             at src/sql/plan_parser.rs:694:18
  16: databend_query::sql::plan_parser::PlanParser::select_to_plan
             at src/sql/plan_parser.rs:513:20
  17: databend_query::sql::plan_parser::PlanParser::query_to_plan
             at src/sql/plan_parser.rs:491:17
  18: databend_query::sql::plan_parser::PlanParser::sql_statement_to_plan
             at src/sql/plan_parser.rs:179:40
  19: databend_query::sql::plan_parser::PlanParser::statement_to_plan
             at src/sql/plan_parser.rs:124:42
  20: databend_query::sql::plan_parser::PlanParser::build_from_sql::{{closure}}::{{closure}}
             at src/sql/plan_parser.rs:100:34
  21: core::option::Option<T>::map
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/option.rs:836:29
  22: databend_query::sql::plan_parser::PlanParser::build_from_sql::{{closure}}
             at src/sql/plan_parser.rs:98:13
  23: core::result::Result<T,E>::and_then
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:966:22
  24: databend_query::sql::plan_parser::PlanParser::build_from_sql
             at src/sql/plan_parser.rs:97:9
  25: databend_query::tests::parse_query::parse_query
             at src/tests/parse_query.rs:23:5
  26: databend_query::optimizers::optimizer_constant_folding_test::tests::test_constant_folding_optimizer
             at src/optimizers/optimizer_constant_folding_test.rs:180:24
  27: databend_query::optimizers::optimizer_constant_folding_test::tests::test_constant_folding_optimizer::{{closure}}
             at src/optimizers/optimizer_constant_folding_test.rs:22:5
  28: core::ops::function::FnOnce::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  29: core::ops::function::FnOnce::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
      test::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:578:5
  30: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
      std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
      test::run_test_in_process
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:601:18
      test::run_test::run_test_inner::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:493:39
  31: test::run_test::run_test_inner::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:520:37
      std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys_common/backtrace.rs:125:18
  32: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:481:17
      <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
      std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
      std::thread::Builder::spawn_unchecked::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:480:30
      core::ops::function::FnOnce::call_once{{vtable.shim}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  33: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys/unix/thread.rs:106:17
  34: __pthread_start

thread 'optimizers::optimizer_constant_folding_test::tests::test_constant_folding_optimizer' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:194:5

---- pipelines::transforms::transform_limit_test::test_transform_limit stdout ----
thread '<unnamed>' panicked at 'UserMgr initialization failure: Code: 1001, displayText = Too many open files (os error 24).

   0: backtrace::backtrace::libunwind::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/libunwind.rs:93:5
      backtrace::backtrace::trace_unsynchronized
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:66:5
      backtrace::backtrace::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:53:14
   1: backtrace::capture::Backtrace::create
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:176:9
      backtrace::capture::Backtrace::new
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:140:22
   2: common_exception::exception::ErrorCode::TokioError
             at /Users/hxl/Desktop/code/myself/databend/common/exception/src/exception.rs:110:77
   3: common_base::runtime::Runtime::create::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:158:36
   4: core::result::Result<T,E>::map_err
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:853:27
   5: common_base::runtime::Runtime::create
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:156:23
   6: common_base::runtime::Runtime::with_worker_threads
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:202:9
   7: common_management::user::user_mgr::UserMgr::new
             at /Users/hxl/Desktop/code/myself/databend/common/management/src/user/user_mgr.rs:47:18
   8: databend_query::users::user_mgr::UserManager::create_global::{{closure}}
             at src/users/user_mgr.rs:47:28
   9: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  10: databend_query::sessions::sessions::SessionManager::from_conf::{{closure}}
             at src/sessions/sessions.rs:60:20
  11: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  12: databend_query::tests::sessions::async_try_create_sessions::{{closure}}
             at src/tests/sessions.rs:26:20
  13: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  14: tokio::park::thread::CachedParkThread::block_on::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:54
  15: tokio::coop::with_budget::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:106:9
  16: std::thread::local::LocalKey<T>::try_with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:399:16
  17: std::thread::local::LocalKey<T>::with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:375:9
  18: tokio::coop::with_budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:99:5
      tokio::coop::budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:76:5
      tokio::park::thread::CachedParkThread::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:31
  19: tokio::runtime::enter::Enter::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/enter.rs:151:13
  20: tokio::runtime::thread_pool::ThreadPool::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/thread_pool/mod.rs:77:9
  21: tokio::runtime::Runtime::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/mod.rs:463:43
  22: databend_query::tests::sessions::sync_try_create_sessions
             at src/tests/sessions.rs:35:5
  23: databend_query::tests::sessions::SessionManagerBuilder::build::{{closure}}
             at src/tests/sessions.rs:101:44
  24: common_base::thread::Thread::spawn::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/thread.rs:37:13
  25: std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys_common/backtrace.rs:125:18
  26: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:481:17
  27: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
  28: std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
  29: <unknown>
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:431:6
  30: std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
  31: std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
  32: std::thread::Builder::spawn_unchecked::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:480:30
  33: core::ops::function::FnOnce::call_once{{vtable.shim}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  34: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys/unix/thread.rs:106:17
  35: __pthread_start
', common/management/src/user/user_mgr.rs:47:50
thread 'pipelines::transforms::transform_limit_test::test_transform_limit' panicked at 'called `Result::unwrap()` on an `Err` value: Any { .. }', query/src/tests/sessions.rs:102:23

---- servers::mysql::mysql_handler_test::test_use_database_with_on_query stdout ----
Error: Code: 1001, displayText = Too many open files (os error 24).

   0: backtrace::backtrace::libunwind::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/libunwind.rs:93:5
      backtrace::backtrace::trace_unsynchronized
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:66:5
      backtrace::backtrace::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:53:14
   1: backtrace::capture::Backtrace::create
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:176:9
      backtrace::capture::Backtrace::new
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:140:22
   2: common_exception::exception::ErrorCode::TokioError
             at /Users/hxl/Desktop/code/myself/databend/common/exception/src/exception.rs:110:77
   3: common_base::runtime::Runtime::create::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:158:36
   4: core::result::Result<T,E>::map_err
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:853:27
   5: common_base::runtime::Runtime::create
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:156:23
   6: common_base::runtime::Runtime::with_worker_threads
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:202:9
   7: <databend_query::servers::mysql::mysql_handler::MySQLHandler as databend_query::servers::server::Server>::start::{{closure}}
             at src/servers/mysql/mysql_handler.rs:137:44
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
   9: <core::pin::Pin<P> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/future.rs:119:9
  10: databend_query::servers::mysql::mysql_handler_test::test_use_database_with_on_query::{{closure}}
             at src/servers/mysql/mysql_handler_test.rs:40:27
  11: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  12: tokio::park::thread::CachedParkThread::block_on::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:54
  13: tokio::coop::with_budget::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:106:9
  14: std::thread::local::LocalKey<T>::try_with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:399:16
  15: std::thread::local::LocalKey<T>::with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:375:9
  16: tokio::coop::with_budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:99:5
      tokio::coop::budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:76:5
      tokio::park::thread::CachedParkThread::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:31
  17: tokio::runtime::enter::Enter::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/enter.rs:151:13
  18: tokio::runtime::thread_pool::ThreadPool::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/thread_pool/mod.rs:77:9
  19: tokio::runtime::Runtime::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/mod.rs:463:43
  20: databend_query::servers::mysql::mysql_handler_test::test_use_database_with_on_query
             at src/servers/mysql/mysql_handler_test.rs:48:5
  21: databend_query::servers::mysql::mysql_handler_test::test_use_database_with_on_query::{{closure}}
             at src/servers/mysql/mysql_handler_test.rs:35:7
  22: core::ops::function::FnOnce::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  23: core::ops::function::FnOnce::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
      test::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:578:5
  24: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
      std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
      test::run_test_in_process
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:601:18
      test::run_test::run_test_inner::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:493:39
  25: test::run_test::run_test_inner::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:520:37
      std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys_common/backtrace.rs:125:18
  26: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:481:17
      <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
      std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
      std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
      std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
      std::thread::Builder::spawn_unchecked::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:480:30
      core::ops::function::FnOnce::call_once{{vtable.shim}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  27: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys/unix/thread.rs:106:17
  28: __pthread_start

thread 'servers::mysql::mysql_handler_test::test_use_database_with_on_query' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/test/src/lib.rs:194:5

---- optimizers::optimizer_expression_transform_test::tests::test_expression_transform_optimizer stdout ----
thread '<unnamed>' panicked at 'UserMgr initialization failure: Code: 1001, displayText = Too many open files (os error 24).

   0: backtrace::backtrace::libunwind::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/libunwind.rs:93:5
      backtrace::backtrace::trace_unsynchronized
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:66:5
      backtrace::backtrace::trace
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/backtrace/mod.rs:53:14
   1: backtrace::capture::Backtrace::create
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:176:9
      backtrace::capture::Backtrace::new
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/backtrace-0.3.62/src/capture.rs:140:22
   2: common_exception::exception::ErrorCode::TokioError
             at /Users/hxl/Desktop/code/myself/databend/common/exception/src/exception.rs:110:77
   3: common_base::runtime::Runtime::create::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:158:36
   4: core::result::Result<T,E>::map_err
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/result.rs:853:27
   5: common_base::runtime::Runtime::create
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:156:23
   6: common_base::runtime::Runtime::with_worker_threads
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/runtime.rs:202:9
   7: common_management::user::user_mgr::UserMgr::new
             at /Users/hxl/Desktop/code/myself/databend/common/management/src/user/user_mgr.rs:47:18
   8: databend_query::users::user_mgr::UserManager::create_global::{{closure}}
             at src/users/user_mgr.rs:47:28
   9: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  10: databend_query::sessions::sessions::SessionManager::from_conf::{{closure}}
             at src/sessions/sessions.rs:60:20
  11: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  12: databend_query::tests::sessions::async_try_create_sessions::{{closure}}
             at src/tests/sessions.rs:26:20
  13: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/future/mod.rs:80:19
  14: tokio::park::thread::CachedParkThread::block_on::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:54
  15: tokio::coop::with_budget::{{closure}}
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:106:9
  16: std::thread::local::LocalKey<T>::try_with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:399:16
  17: std::thread::local::LocalKey<T>::with
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/local.rs:375:9
  18: tokio::coop::with_budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:99:5
      tokio::coop::budget
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/coop.rs:76:5
      tokio::park::thread::CachedParkThread::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/park/thread.rs:263:31
  19: tokio::runtime::enter::Enter::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/enter.rs:151:13
  20: tokio::runtime::thread_pool::ThreadPool::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/thread_pool/mod.rs:77:9
  21: tokio::runtime::Runtime::block_on
             at /Users/hxl/.cargo/registry/src/mirrors.sjtug.sjtu.edu.cn-7a04d2510079875b/tokio-1.12.0/src/runtime/mod.rs:463:43
  22: databend_query::tests::sessions::sync_try_create_sessions
             at src/tests/sessions.rs:35:5
  23: databend_query::tests::sessions::SessionManagerBuilder::build::{{closure}}
             at src/tests/sessions.rs:101:44
  24: common_base::thread::Thread::spawn::{{closure}}
             at /Users/hxl/Desktop/code/myself/databend/common/base/src/thread.rs:37:13
  25: std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys_common/backtrace.rs:125:18
  26: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:481:17
  27: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/panic/unwind_safe.rs:271:9
  28: std::panicking::try::do_call
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:403:40
  29: <unknown>
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:431:6
  30: std::panicking::try
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panicking.rs:367:19
  31: std::panic::catch_unwind
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/panic.rs:129:14
  32: std::thread::Builder::spawn_unchecked::{{closure}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/thread/mod.rs:480:30
  33: core::ops::function::FnOnce::call_once{{vtable.shim}}
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/core/src/ops/function.rs:227:5
  34: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/alloc/src/boxed.rs:1636:9
      std::sys::unix::thread::Thread::new::thread_start
             at /rustc/b69fe57261086e70aea9d5b58819a1794bf7c121/library/std/src/sys/unix/thread.rs:106:17
  35: __pthread_start
', common/management/src/user/user_mgr.rs:47:50
thread 'optimizers::optimizer_expression_transform_test::tests::test_expression_transform_optimizer' panicked at 'called `Result::unwrap()` on an `Err` value: Any { .. }', query/src/tests/sessions.rs:102:23


failures:
    optimizers::optimizer_constant_folding_test::tests::test_constant_folding_optimizer
    optimizers::optimizer_expression_transform_test::tests::test_expression_transform_optimizer
    pipelines::transforms::transform_limit_test::test_transform_limit
    servers::mysql::mysql_handler_test::test_rejected_session_with_sequence
    servers::mysql::mysql_handler_test::test_use_database_with_on_query

test result: FAILED. 131 passed; 5 failed; 0 ignored; 0 measured; 0 filtered out; finished in 6.30s

error: test failed, to rerun pass '-p databend-query --lib'
make: *** [unit-test] Error 101
```

--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,7 @@ cli-test:
 	bash ./scripts/ci/ci-run-cli-unit-tests.sh
 
 unit-test:
-	bash ./scripts/ci/ci-run-unit-tests.sh
+	ulimit -c 1024 bash ./scripts/ci/ci-run-unit-tests.sh
 
 embedded-meta-test: build-debug
 	rm -rf ./_meta_embedded
@@ -69,7 +69,7 @@ embedded-meta-test: build-debug
 
 stateless-test: build-debug
 	rm -rf ./_meta/
-	bash ./scripts/ci/ci-run-stateless-tests-standalone.sh
+	ulimit -c 1024 bash ./scripts/ci/ci-run-stateless-tests-standalone.sh
 
 stateless-cluster-test: build-debug
 	rm -rf ./_meta/

